molecule/additional_flags/molecule.yml

```diff
@@ -0,0 +1 @@
+molecule-with-systemd.yml
```

Comment:
Should this be set up like the one for `molecule/modify_config`?

Reply:
Please see commit 2e1b063.